packages/components/notification/src/Notification.tsx

```diff
@@ -54,20 +54,20 @@ function afterInit<PromiseValueType>(fn: Function) {
   };
 }
 
-const show = (intent: NotificationIntent) => (
+const show = (variant: NotificationVariant) => (
   text: string,
   settings?: {
     duration?: number;
-    canClose?: boolean;
+    withClose?: boolean;
```

Comment:
also, a questionable change for me. I'd rather not change it. It's not a component, but a part of an imperative API. IMO, it even better than `withClose`

Reply:
Yeah, also updated this one to follow what is set on the guidelines that when a prop should represent if a nested component should be rendered it should use the `with` prefix.
But considering that this is more and API than a component I'm fine with going back to `canClose`

Reply:
let's discuss with the team

Reply:
For me, is the same thing here that @denkristoffer said about the other prop

v4 is a chance to align things with breaking changes
if we don't break now, we will need v5 to do it and when we work in v5 we will probably have this discussion again hahahah
and in v5 we also won't break things so we will do in v6

you see where I'm going with this

Reply:
I disagree with renaming this property. Why do we need to do a breaking change here? `canClose` is actually describing the intention better... Can the user close this notification?

Reply:
I agree that the `canClose` describes better the intention, and also this won't be used as a prop by the user itself, it will be passed through the settings when calling a show method from `Notification`.

Reply:
I don't have a strong opinion on it. Either way seems fine for me. So I'd rather go forward with the approach that needs a least amount of work.

Reply:
I'd root for leaving as it is now.